modules/cluster-tagging.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+// * machine_management/more-rhel-compute.adoc
+
+
+[id="cluster-tagging_{context}"]
+= Tagging an existing cluster as owned
+
+A cluster uses the value of the `kubernetes.io/cluster/<clusterid>,Value=(owned|shared)` tag to determine which resources belong to the AWS cluster. These resources include:
+
+* All hosts.
+* All relevant load balancers to be used in the AWS instances.
+* All relevant security groups to be used with the AWS instances.
+* All EBS volumes.
++
+[TIP]
+====
+Run the following command to find all EBS volumes that must be tagged:
+[source,terminal]
+----
+$ oc get pv -o json|jq '.items[].spec.awsElasticBlockStore.volumeID'
+----
+====
+
+.Procedure
+
+. With {op-system-base} compute machines, set the cluster to owned with the `kubernetes.io/cluster/<clusterid>=owned` tag using the appropriate cluster ID.
```

Comment:
I think adding tag to RHEL machine should before [this step](https://deploy-preview-31600--osdocs.netlify.app/openshift-enterprise/latest/machine_management/adding-rhel-compute.html#rhel-adding-node_adding-rhel-compute), otherwise the command `ansible-playbook` will fail.

A good option is, adding tag when you creating RHEL instance.

Reply:
Would this be a "Prerequisite" item I add to the step?

Reply:
since this is documentation for adding a single RHEL worker, let narrow the scope of the instruction to "the RHEL worker instance needs be tagged with `kubernetes.io/cluster/<clusterid>=owned`,

Reply:
> Would this be a "Prerequisite" item I add to the step?

yes

Reply:
@sjenning is there any particular reason that need to tag host with `owned`? thanks!
as my [above comment](https://github.com/openshift/openshift-docs/pull/31600#discussion_r619742350), I think it should be left to the user to choose.